dev/ProgressBar/ProgressBar.cpp

```diff
@@ -121,7 +123,16 @@ void ProgressBar::SetProgressBarIndicatorWidth()
 
             if (IsIndeterminate())
             {
-                progressBarIndicator.Width(progressBarWidth * 0.4);
+                determinateProgressBarIndicator.Width(0);
```

Comment:
>determinateProgressBarIndicator.Width(0); [](start = 16, length = 41)

Is this needed? we set it's opacity to 0 in the VSM right?

Reply:
@StephenLPeters It seemed to still be visible during the first frame of each animation if the width is non-zero. Furthermore, this allows a smoother animation as it switches from indeterminate mode back to determinate.

Reply:
@StephenLPeters the indicator seems to appear for a split second at the beginning of each animation. This also allows a smoother animation switching from indeterminate back to determinate.